src/toolkits/style_transfer/style_transfer.cpp

```diff
@@ -347,6 +353,92 @@ void style_transfer::infer_derived_options() {
   add_or_update_state({{"training_iterations", 0}});
 }
 
+gl_sframe style_transfer::get_styles(variant_type style_index) {
+  gl_sframe style_sf = read_state<gl_sframe>("styles");
+  gl_sarray style_filter = convert_style_indicies_to_filter(style_index);
+
+  return style_sf[style_filter];
+}
+
+gl_sframe style_transfer::style_sframe_with_index(gl_sarray styles) {
+  std::vector<flexible_type> index_list;
+  flex_int num_styles = read_state<flex_int>("num_styles");
+  index_list.resize(num_styles);
+  std::iota(index_list.begin(), index_list.end(), 0);
+
+  return gl_sframe({
+      {"index", index_list},
+      {"style", styles},
+  });
+}
+
+/**
+ * convert_style_indicies_to_filter
+ *
+ * This function takes a variant type and converts it into a boolean filter. The
+ * elements at the indicies we want to keep are set to a value of `1`, the
+ * elements we don't want to keep are set to a value of `0`.
+ */
+gl_sarray style_transfer::convert_style_indicies_to_filter(const variant_type& data) {
```

Comment:
The plural of "index" is "indices" (or "indexes", as in some Apple APIs)

Reply:
Thanks for catching the spelling mistakes!